native_client/python/__init__.py

```diff
@@ -95,6 +95,22 @@ def disableExternalScorer(self):
         """
         return deepspeech.impl.DisableExternalScorer(self._impl)
 
+    def addHotWord(self, word, boost):
```

Comment:
This does looks good, but can you make sure you expose also this on JS and Java side? It should be pretty straight forward .Net could be cool but @carlfm01 maybe can help there.

Reply:
did this in recent commits